packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1510,10 +1503,13 @@ export function _useFakeChatAdapters(args: _FakeChatAdapterArgs): _FakeChatAdapt
 export const useTeamsCallAdapter: (args: Partial<TeamsCallAdapterArgs>, afterCreate?: ((adapter: TeamsCallAdapter) => Promise<TeamsCallAdapter>) | undefined, beforeDispose?: ((adapter: TeamsCallAdapter) => Promise<void>) | undefined) => TeamsCallAdapter | undefined;
 
 // @beta
-export interface VideoBackgroundBlurEffect {
+export interface VideoBackgroundBlurEffect extends BackgroundBlurConfig {
     effectName: 'blur';
 }
 
+// @beta
+export type VideoBackgroundEffect = VideoBackgroundNoEffect | VideoBackgroundBlurEffect | VideoBackgroundReplacementEffect;
```

Comment:
Is this 3 types same as what comes from sdk?

Reply:
No these are what we are defining, but they do extend the types coming from sdk `BackgroundBlurConfig` and `BackgroundReplacementConfig`
<img width="634" alt="Screenshot 2023-06-07 at 3 05 28 PM" src="https://github.com/Azure/communication-ui-library/assets/97124699/5f373164-bdc9-4ad9-9f1a-8521685880b8">